components/device-mgt/org.wso2.carbon.device.mgt.ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.view/public/css/main.css

```diff
@@ -142,6 +142,7 @@
 }
 .tab-actions {
 	margin: 0px;
+    margin-bottom: 10px;
```

Comment:
Line indentation formatting issue, please fix this.

Reply:
Fixed in https://github.com/wso2/carbon-device-mgt/pull/950